036-BGP/README.md

```diff
@@ -0,0 +1,64 @@
+# What The Hack - Using BGP for Hybrid Connectivity
+
+## Introduction
+
+In this Hack exercise students will explore how to use BGP in a complex environment between an on-premises network and two Azure regions. Both active/passive and active/active Azure VPN Gateways will be used in Azure, and Cisco CSR 1000v will be used to simulate onprem devices.
+
+The challenges will show real-world scenarios that have been observed in customers deploying their applications to Azure
+
+The estimated duration time for this hack is 1.5-2 days.
+
+The only cost for this Hack is Azure infrastructure, there are no licensing costs associated to the Cisco NVAs.
+
+## Learning Objectives
+
+After completing this Hack, participants will be familiar with these concepts, amongst others:
+
+- BGP
+- Differences between eBGP and iBGP
+- Understanding Autonomous Systems
+- Using AS-path prepending to influence routing
+- Explore BGP route manipulation capabilities
+- BGP communities
+
+## Before you start
+
+Please read these instructions carefully:
+
+- Since this Hack is not about deploying VNets or NVAs, you can use a script to deploy the infrastructure that you will be working on. You will find a script called `bgp.sh` in the file resources supplied for this Hack, which you can run to deploy the environment. The script has been tested to run in Azure Cloud Shell. Note that the script takes around 1 hour to complete, and it requires certain dependencies to exist. Therefore, it is recommended to deploy the environment the day before the Hack:
+- Setting expectations: all of the challenges in this Hack would take around 2 days to complete. Please do not expect to finish all of the exercises in a shorter event
+- It is recommended going one challenge after the other, without skipping any. However, if your team decides to modify the challenge order, that is possible too. Please consult with your coach to verify that the challenge order you wish to follow is doable, and there are no dependencies on the challenges you skip
+- **Think** before rushing to configuration. One minute of planning might save you hours of work
+- Look for the **relevant information** section in each challenge, they might contain useful information and tools
+- You might want to split the individual objectives of a challenge across team members, but please consider that all of the team members need to understand every part of a challenge, so run a retrospective after each subteam has finished and share lessons learnt
+
+These are your challenges, it is recommended to start with the first one and proceed to the next one when your coach confirms that you have completed each challenge successfully:
+
+## Challenges
+
+- Challenge 1: **[Environment Setup](Student/01-lab_setup.md)**
```

Comment:
Not sure if we already mentioned it, but typically WTHs use "Challenge 0" for environment setup tasks. I can't remember how we left it.  If it is more hassle than it's worth, no worries, leave it as is.

Reply:
I was thinking about that. However, it is not just setting up the environment, there is an exploration component to it to learn to navigate the Azure and 3rd-party components in the lab. I would suggest to leave it as challenge 1, if you agree.

Reply:
I finally decided to listen to  you :-)  I split my challenge 1 into two: challenge 0 (setup) and challenge 1 (exploration)